internal/flow/promote.go

```diff
@@ -46,6 +46,14 @@ func (s *Service) Promote(ctx context.Context, actor Actor, environment, namespa
 			return true, errors.WithMessage(err, fmt.Sprintf("locate source spec"))
 		}
 
+		ok, err := s.ReleaseRestrictor(ctx, service, sourceSpec.Application.Branch, environment)
```

Comment:
The wording here makes kind of hard to read. Could it be something like `CanRelease` to avoid double negatives?
Also how is errors here actually handled? Seems like the error should be `"failed reading release restriction"`?

Reply:
Good point. That would make it more clear 👍

Reply:
Renamed to `CanRelease` and also updated the error wrapping messages. 👍